tests/infra/jsonrpc.py

```diff
@@ -282,15 +283,16 @@ def __init__(
         self.name = "[{}:{}]".format(host, port)
         self.description = description
         self.rpc_loggers = (RPCLogger(),)
+        self.prefix = prefix
 
     def connect(self):
         return self.client.connect()
 
     def disconnect(self):
         return self.client.disconnect()
 
-    def request(self, *args, **kwargs):
-        r = self.stream.request(*args, **kwargs)
+    def request(self, method, params, *args, **kwargs):
```

Comment:
what happens here if `prefix` is `None`?

Reply:
This works, producing a method like `"None/getCommit"`, which will produce an error later. I've replaced the default prefixes from `None` to `"users"` to mitigate this.